single-line-diagram/single-line-diagram-core/src/test/java/com/powsybl/sld/iidm/TestFeederInfos.java

```diff
@@ -136,7 +136,7 @@ void testFrenchFormatting() {
         voltageLevelGraphLayout(g);
 
         // write SVG and compare to reference
-        assertEquals(toString("/TestFormattingFeederInfos.svg"), toSVG(g, "/TestFormattingFeederInfos.svg"));
+        assertEquals(toString("/TestFormattingFeederInfosNNBSP.svg"), toSVG(g, "/TestFormattingFeederInfos.svg"));
```

Comment:
Since we finally have only one version of the reference file, we can keep its original name (`TestFormattingFeederInfos.svg`).